nautobot_data_validation_engine/models.py

```diff
@@ -366,3 +367,65 @@ def clean(self):
 
         if getattr(model_field, "unique", False):
             raise ValidationError({"field": "This field is already unique by default."})
+
+
+class AuditRule(PrimaryModel):
+    """Model to represent the results of a audit method."""
+
+    class_name = models.CharField(max_length=100, blank=False, null=False)
```

Comment:
No issues with explicitly defining them, but just to point out, both `blank` and `null` both default to `False`.